setup.py

```diff
@@ -26,15 +26,45 @@
 
 
 extras_require = {
+    'client': [
+        'netifaces',
+        'posttroll>=1.5.1',
+        'pyinotify',
+        'pyzmq',
+        'trollsift',
+    ],
+    'dispatcher': [
```

Comment:
dispatcher needs posttroll too to receive messages

Reply:
Fixed in https://github.com/pytroll/trollmoves/pull/129/commits/f1c5fbfc41b2935e8a0be701a4b608b1cf2eadf4